script.js

```diff
@@ -25,7 +25,7 @@ var config = {
     },
     scene: {
         preload: preload,
-        create: createLevel1, //Temp
+        create: createLevel4, //Temp
```

Comment:
Looks like you forgot to take this out after testing. We have to start at level 1! Remove the comment while you're at it.